### ISSUE TITLE
clj-kondo reporting an error on line 0

### ISSUE THREAD

**Issue by marcomorain** [borkdude/clj-kondo#497]
**version**

`clj-kondo v2019.09.22-alpha`

**platform**

```
$ uname -a
Darwin blaster.local 18.7.0 Darwin Kernel Version 18.7.0: Tue Aug 20 16:57:14 PDT 2019; root:xnu-4903.271.2~2/RELEASE_X86_64 x86_64
```

**problem**

`clj-kondo` is reporting a problem on line 0 of a file. This is causing an issue for `clojure-lint`

```
$ echo '(ns circleci.rollcage.test-core (:require [clojure.test :refer :refer [deftest]]))' | clj-kondo --lint -
<stdin>:0:0: error: can't parse <stdin>, No implementation of method: :sexpr of protocol: #'clj-kondo.impl.rewrite-clj.node.protocols/Node found for class: nil
linting took 11ms, errors: 1, warnings: 0
```

**expected behavior**

I would expect the line number to be greater than zero.

**actual behavior**

```
$ clj-kondo --lint test
test/circleci/rollcage/test_core.clj:0:0: error: can't parse test/circleci/rollcage/test_core.clj, No implementation of method: :sexpr of protocol: #'clj-kondo.impl.rewrite-clj.node.protocols/Node found for class: nil
```
